### ISSUE TITLE
WARN StateMonitor:4e4c65886b652b0b:Rank1 : kernel begins at work 55731 may hang in polling, time 20250710-204243

### ISSUE THREAD

**Issue by sincerity-being** [MoonshotAI/Kimi-Audio#133]
: using eth0:22.1.6.167<55133>
[dsw-1074031-5b7b46d667-rqd5s][3598465:3601782][0][state_monitor.cc:89] NCCL INFO StateSync:4e4c65886b652b0b:Rank2 : using eth0:22.1.6.167<54911>
{'loss': 10.2027, 'grad_norm': 219.71975217379563, 'learning_rate': 0.0, 'epoch': 0.0}                             
{'loss': 10.6178, 'grad_norm': 212.43534775318054, 'learning_rate': 1.9823986317056056e-06, 'epoch': 0.0}          
{'loss': 14.8933, 'grad_norm': 267.0969388071493, 'learning_rate': 3.1420274927343152e-06, 'epoch': 0.0}           
{'loss': 17.8956, 'grad_norm': 307.1107972382939, 'learning_rate': 3.964797263411211e-06, 'epoch': 0.0}            
{'loss': 10.7146, 'grad_norm': 215.4441432819491, 'learning_rate': 4.6029870782235105e-06, 'epoch': 0.0}           
{'loss': 9.3495, 'grad_norm': 214.68268382981034, 'learning_rate': 5.124426124439921e-06, 'epoch': 0.0}            
{'loss': 8.3231, 'grad_norm': 168.3308851377049, 'learning_rate': 5.5652965561989914e-06, 'epoch': 0.0}            
{'loss': 4.4251, 'grad_norm': 174.2487709718774, 'learning_rate': 5.9471958951168165e-06, 'epoch': 0.0}            
{'loss': 9.638, 'grad_norm': 121.01012248059192, 'learning_rate': 6.2840549854686305e-06, 'epoch': 0.01}           
{'loss': 4.0833, 'grad_norm': 97.28903168853292, 'learning_rate': 6.5853857099291174e-06, 'epoch': 0.01}           
{'loss': 5.3412, 'grad_norm': 89.65712279949317, 'learning_rate': 6.857972507265687e-06, 'epoch': 0.01}            
{'loss': 7.3222, 'grad_norm': 93.933179806271, 'learning_rate': 7.106824756145526e-06, 'epoch': 0.01}              
{'loss': 3.9767, 'grad_norm': 79.96081385684437, 'learning_rate': 7.335746633951978e-06, 'epoch': 0.01}            
{'loss': 5.5749, 'grad_norm': 75.1276481362465, 'learning_rate': 7.547695187904597e-06, 'epoch': 0.01}             
{'loss': 8.7092, 'grad_norm': 103.59520289549621, 'learning_rate': 7.745014570957826e-06, 'epoch': 0.01}           
  0%|▎                                                                         | 15/3206 [02:26<8:23:31,  9.47s/it]
[dsw-1074031-5b7b46d667-rqd5s][3598465:3599084][0][state_monitor.cc:219] NCCL WARN StateMonitor:679406a81420dd02:Rank2 : kernel begins at work 40175 may hang in polling, time 20250710-204218

[dsw-1074031-5b7b46d667-rqd5s][3598465:3599085][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank2 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598463:3599087][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank0 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598464:3599086][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank1 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598464:3601779][0][state_monitor.cc:219] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank1 : kernel begins at work 26718 may hang in polling, time 20250710-204220

[dsw-1074031-5b7b46d667-rqd5s][3598464:3601781][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank1 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598463:3601780][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank0 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598465:3601782][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank2 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598463:3601778][0][state_monitor.cc:219] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank0 : kernel begins at work 26718 may hang in polling, time 20250710-204223

[dsw-1074031-5b7b46d667-rqd5s][3598463:3601778][0][state_monitor.cc:219] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank0 : kernel begins at work 55731 may hang in polling, time 20250710-204235

[dsw-1074031-5b7b46d667-rqd5s][3598463:3601780][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank0 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598465:3601782][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank2 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598465:3599084][0][state_monitor.cc:219] NCCL WARN StateMonitor:679406a81420dd02:Rank2 : kernel begins at work 51312 may hang in polling, time 20250710-204236

[dsw-1074031-5b7b46d667-rqd5s][3598465:3599085][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank2 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598463:3599087][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank0 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598464:3601781][0][state_monitor.cc:122] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank1 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598464:3599086][0][state_monitor.cc:122] NCCL WARN StateMonitor:679406a81420dd02:Rank1 : trigger debugging dump

[dsw-1074031-5b7b46d667-rqd5s][3598464:3601779][0][state_monitor.cc:219] NCCL WARN StateMonitor:4e4c65886b652b0b:Rank1 : kernel begins at work 55731 may hang in polling, time 20250710-204243
15 step stop training

**Comment by shyringo** [comment]
same problem. Does it mean the process has been stuck? 
Really wondering how to fix the problem.

**Comment by TKL-tao** [comment]
the same problem
